nejma/layers.py

```diff
@@ -73,12 +78,73 @@ def validate_name(self, name):
             + "only alphanumerics and underscores are accepted"
         )
 
-    def clean_expired(self):
+    async def clean_expired(self):
         for group in self.groups:
             # Can't change dict size during iteration
             for channel in list(self.groups.get(group, {})):
                 if channel.is_expired():
                     del self.groups[group][channel]
 
 
+class RedisLayer(ChannelLayer):
+    def __init__(self, redis_host="redis://localhost", prefix="group"):
```

Comment:
Can you add an assert to check if `aioredis` is installed and some message e.g. `assert aioredis is not None, "aioredis must be installed to use RedisLayer"`

Reply:
Yeah, I first thought that aioredis defaulting to None would be ok.
But I would use exception, maybe ValueError? or AssertionError, as asserts aren't good in production

Reply:
Done

Reply:
Assertion error is raised